examples/managed-identity/kubernetes.json

```diff
@@ -0,0 +1,38 @@
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes",
+      "kubernetesConfig": {
+        "useManagedIdentity": true

```

Comment:
Very interesting question. In the future, there will be explicit identity support, meaning that users can pre-provision AAD SPs as real resources (ie `/Microsoft.Identity/explicitIdentities/k8s-master0`).

I think that can be an additive change, so it might look like this in the future:

```
... {
  "orchestratorType": "Kubernetes",
  "kubernetesConfig": {
     "useManagedIdentity": true,
     "masterIdentity": "/.../Microsoft.Identity/master-sp",
     "nodeIdentity": "/.../Microsoft.Identity/node-sp",
   }
} ...
```